Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -30,6 +55,13 @@ public static void Reverse(int[] array)
         public static int[] RemoveLast(int[] array)
         {
             // ToDo: implement.
+            if(array != null)
+            {
+                int[] newArr = new int[array.Length-1];
+                for (int i = 0; i < array.Length - 1; ++i)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'for' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4940124)